network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/NetworkImpl.java

```diff
@@ -48,7 +48,7 @@ public static NetworkImpl create(NetworkStoreClient storeClient, Resource<Networ
 
         @Override
         public Stream<Bus> getBusStream() {
-            return getVoltageLevelStream().flatMap(vl -> vl.getBusBreakerView().getBusStream());
+            return getVoltageLevelStream().filter(vl -> vl.getTopologyKind() == TopologyKind.BUS_BREAKER).flatMap(vl -> vl.getBusBreakerView().getBusStream());
```

Comment:
You add a regression here, we want to iterate on all voltage levels bus breaker view including bus/breaker view on node/breaker voltage level. So we do not have to filter on voltage level topology kind.